projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss

```diff
@@ -211,3 +211,54 @@ $_bootstrap-button-group: extend(
     )
 );
 
+$_indigo-button-group: extend(
+    $_light-button-group,
+    $_indigo-shape-button-group,
+    $_indigo-elevation-button-group,
+    (
+        variant: 'indigo-design',
+
+        item-background: (
+            igx-color: ('primary', 500),
+        ),
+
+        item-text-color: #fff,
+
+        item-border-color: (
+            igx-color: ('primary', 400),
+        ),
+
+        item-hover-background: (
+            igx-color: ('primary', 400),
+        ),
+
+        item-hover-text-color: #fff,
+
+        item-selected-background: (
+            igx-color: ('primary', 400)
+        ),
+
+        item-selected-text-color: #fff,
+
+        item-selected-border-color: (
+            igx-color: ('primary', 400),
+        ),
+
+        item-selected-hover-background: (
+            igx-color: ('primary', 400)
+        ),
+
+        disabled-background-color: (
+            igx-color: ('primary', 500),
+            rgba: .4
+        ),
+
+        disabled-text-color: #fff,
+
+        item-disabled-border: (
+            igx-color: ('primary', 400),
+            rgba: .4
+        )
+    )
```

Comment:
I am not really sure about those disabled colors. Especially the bright white that's used for text. Plus, the button group and the button differ too much in that respect.

Reply:
@StefanIvanov

Reply:
```suggestion
        disabled-background-color: (
            igx-color: ('grays', 100)
        ),

        disabled-text-color: (
            igx-color: ('grays', 400)
        ),

        item-disabled-border: (
            igx-color: ('grays', 100)
        ),
    )
```

Reply:
These look a lot better.